app/src/main/java/org/projectbuendia/client/sync/SyncAdapter.java

```diff
@@ -267,6 +268,7 @@ public SyncAdapter(Context context, boolean autoInitialize, boolean allowParalle
                     // there are only a few forms, usually less than 10.
                     case SYNC_FORMS:
                         updateForms(provider, syncResult);
+                        OdkActivityLauncher.fetchAndCacheAllXforms();
```

Comment:
Here i was in doubt if `OdkActivityLauncher.fetchAndCacheAllXforms();` should be inside the `updateForms` method.
But i' rather to keep them isolated given they do different things.